pkg/syncer/metadatasyncer.go

```diff
@@ -168,6 +170,22 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		metadataSyncer.volumeManager = volumes.GetManager(ctx, vCenter)
 	}
 
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
+		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSISVFeatureStateReplication) {
```

Comment:
Don't we want to check if it is a seed-shoot cluster here? Are we planning to use this technique for TKGS in default as well?

Reply:
no need to check the seed or shoot cluster here. we need to start FSS replication service in the supervisor csi controller pod.